portfolio/src/main/webapp/comments-script.js

```diff
@@ -16,28 +16,37 @@
  * Fetches the comments input and builds the UI into a list.
  */
 
+var row;
 function getComments() {
-    const responsePromise = fetch('/comments');
-    responsePromise.then(handleResponse);
+    const RESPONSE_PROMISE = fetch('/comments');
```

Comment:
I don't think you need to use UPPER_SNAKE_CASE for these variables. They are constant (can't be re-assigned to) but the rule for UPPER_SNAKE_CASE is that the type should be deeply immutable and statically (class) scoped.

Reply:
See https://google.github.io/styleguide/jsguide.html#naming-constant-names